ieeg_motorMiller2007/sub-gf/ses-01/ieeg/sub-gf_ses-01_task-motor_run-01_channels.tsv

```diff
@@ -0,0 +1,63 @@
+name	type	sampling_frequency	units	low_cutoff	high_cutoff	notch	reference	status
+1	ECOG	1000	microV	200	0.15	n/a	scalp	good
```

Comment:
Why are all the reverences missing values?

Reply:
by this do you mean "why is it not `n/a`"? or "why is that value not present at all?"

Reply:
I did not know `notch` column is compulsory. All is fine!